eclipse-scout-core/src/calendar/Calendar.ts

```diff
@@ -2317,6 +2328,14 @@ export class Calendar extends Widget implements CalendarModel {
     }
   }
 
+  /**
+   * Removes range selection and triggers a selectedRangeChange event
+   */
+  removeRangeSelection() {
```

Comment:
This does only remove the selection in the DOM but the model state won't change. It will send selectedRangeChange events with the old range to the server. Better use _setSelectedRange instead, you can make it public